models/pull.js

```diff
@@ -220,6 +221,7 @@ Pull.getFromDB = function(data, signatures, comments, reviews, commitStatuses, l
       state: data.state,
       title: data.title,
       body: data.body,
+      draft: data.draft === 1,
```

Comment:
Just a JS tip: `0` is `false` in JS, and non-zero numbers are `true`. So you could use

```suggestion
      draft: Boolean(data.draft),
```

or traditionally

```js
      draft: !!data.draft,
```
(`!` results in a boolean value; double-negation keeps the boolean the same; it's a idiomatic trick)

Reply:
True, I advised the `=== 1` to be really clear what data was expected from the DB.